From c773b27db829a6757ebd2b069e3f69162a6db5d6 Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Mon, 10 Jul 2023 08:47:08 -0500
Subject: [PATCH] Fix install location for assets and shaders

This fixes the following error for the cross-compile case, where
RESOURCE_INSTALL_DIR=/usr/share/vulkan-demos:
```
Error: Could not open shader file \"/usr/share/vulkan-demos/shaders/glsl/renderheadless/triangle.frag.spv\"
```

Upstream-Status: Submitted [https://github.com/SaschaWillems/Vulkan/pull/1049]

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a58b0227..907f057b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,8 +121,8 @@ endfunction(buildExamples)
 if(RESOURCE_INSTALL_DIR)
 	add_definitions(-DVK_EXAMPLE_ASSETS_DIR=\"${RESOURCE_INSTALL_DIR}/\")
 	add_definitions(-DVK_EXAMPLE_SHADERS_DIR=\"${RESOURCE_INSTALL_DIR}/\")
-	install(DIRECTORY assets/ DESTINATION ${RESOURCE_INSTALL_DIR}/)
-	install(DIRECTORY shaders/ DESTINATION ${RESOURCE_INSTALL_DIR}/)
+	install(DIRECTORY assets/ DESTINATION ${RESOURCE_INSTALL_DIR}/assets/)
+	install(DIRECTORY shaders/ DESTINATION ${RESOURCE_INSTALL_DIR}/shaders/)
 else()
 	add_definitions(-DVK_EXAMPLE_ASSETS_DIR=\"${CMAKE_SOURCE_DIR}/assets/\")
 	add_definitions(-DVK_EXAMPLE_SHADERS_DIR=\"${CMAKE_SOURCE_DIR}/shaders/\")
-- 
2.25.1

